scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd "${LAZYDOCKER_HOME}" || exit 1
+  if doIsWindows
+  then
+    ldInPathRes=$(echo "$PATH" | tr ":" "\n" | grep lazydocker)
+    if [ -z "${ldInPathRes}" ]
+    then
+      export PATH=${PATH}:"${LAZYDOCKER_HOME}"
+    fi
```

Comment:
This seems to be a missunderstanding.
If `doSetup` was called then `lazydocker` has to be installed and on the `PATH`. Otherwise this is a bug of `doSetup`.
IMHO we do not need this block.